po/fr.po

```diff
@@ -2062,12 +2062,18 @@ msgstr ""
 "                        voyez les fichiers de configuration dans le répertoire de configuration"
 
 #: cobc/help.c:82
-msgid "  -F, -free             use free source format"
-msgstr "  -F, -free             utiliser le format source libre"
+msgid ""
```

Comment:
The idea is good, but we don't do the changes directly, they would be overwritten.

Please target any updates to https://translationproject.org/domain/gnucobol.html. As those are new / changed msgids the translation will need to wait until the release candidate (End of August) is done and the new gnucobol.pot is sent to the TP (at this time we'll pull all current translations, then let the files be recreated, send the msgid file to TP - and before the release pull the translations again).

Reply:
Ok noted. I was not really aware of that process for translations. Thanks for the info :smile: